app/views/posts/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Hi! I&apos;m a show an specific post for an specific user page</h1>
\ No newline at end of file
+<main>
```

Comment:
- [ ] Good Job the view and the app are working but please make sure that your design follows the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAAB43NGWUKMN6BZNMAUSFR7ACY6UV6UQ):- 
- first in /users route , please make sure that the background is white as in the design and the width of the  user cards looks exactly like the one given in the design
 
| Your design  | wireframe |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/215242172-0ab799b7-71b1-4940-9154-5daa2f5fe7af.png)  | ![image](https://user-images.githubusercontent.com/74744787/215242159-ddfe4b21-b556-4c87-bf5f-3605470cc29d.png)  |
- in user page make sure the background is white 
also make sure to add comments to last three posts as in the design.


| Your design  | wireframe |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/215242362-bc65b0b4-aa69-4055-a804-9045a941951c.png)  | ![image](https://user-images.githubusercontent.com/74744787/215242348-662b39ff-5463-47b5-b135-2dd2a2368a7e.png)  |
- in the post section make sure that you provide a count for the comments  as in the design and make sure that the background is white to match the design.

| Your design  | wireframe |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/215242419-13c09a28-c511-4ebb-9f21-e60247706bc4.png) | ![image](https://user-images.githubusercontent.com/74744787/215242437-40d2ea36-04a2-4b27-a6e4-ba4db2379e03.png) |

Reply:
Changes done! 😄